__tests__/endpoints.test.js

```diff
@@ -0,0 +1,61 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const db = require("../db/connection");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  describe("GET", () => {
+    test("status: 200 - responds with an array of all the topic objects.", () => {
+      return request(app).get("/api/topics").expect(200);
+    });
+    test("status: 404 - responds with a 404 if route does not exist.", () => {
+      return request(app)
+        .get("/api/topic/")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Error 404 - Route not found");
+        });
+    });
+  });
+});
+
+describe("/api/articles/:article_id", () => {
+  describe("GET", () => {
+    test("status: 200 - responds with the specified article.", () => {
```

Comment:
Your declaring that you're testing for a specified article, but the test just checks for ANY object that matches those matchers. 

If you're after a specific article, you can out the data you expect to get back from that article. 
At the moment you could be getting the wrong article, and your test wouldnt show this.